ui/widgets/nuxeo-operation-button.js

```diff
@@ -235,6 +283,87 @@ import '../actions/nuxeo-action-button-styles.js';
     _computeTooltip(tooltip, label) {
       return tooltip || label;
     }
+
+    _isPageProviderDisplayBehavior(input) {
+      return (
+        input &&
+        input.behaviors &&
+        Nuxeo.PageProviderDisplayBehavior &&
+        Nuxeo.PageProviderDisplayBehavior.every((p) => input.behaviors.includes(p))
+      );
+    }
+
+    _isSelectAllActive(input) {
+      return this._isPageProviderDisplayBehavior(input) && input.selectAllActive;
+    }
+
+    _isRunning(status) {
+      if (status['entity-type'] === 'bulkStatus') {
+        const { state } = status.value ? status.value : status;
+        return state !== 'ABORTED' && state !== 'COMPLETED';
+      }
+      return status === 'RUNNING';
+    }
+
+    _isAborted(status) {
+      if (status['entity-type'] === 'bulkStatus') {
+        const { state } = status.value ? status.value : status;
+        return state === 'ABORTED';
+      }
+      return this._isRunning(status);
+    }
+
+    _hasBulkStatus(status) {
+      return status['entity-type'] === 'bulkStatus';
+    }
+
+    _onPollStart(e) {
+      if (!e.detail || !e.detail.commandId) {
+        return;
+      }
+      const { commandId } = e.detail;
+      const detail = {
+        message: this.i18n('operationButton.bulk.poll.scheduled', this.i18n(this.label)),
+        abort: function() {
+          this.$.op._abort(commandId);
+        }.bind(this),
+        dismissible: true,
+        sticky: true,
```

Comment:
What does sticky do? I guess we should document all of these new properties somewhere in code (perhaps on notify behavior?) and in the documentation site as well (see [NXDOC-2334](https://jira.nuxeo.com/browse/NXDOC-2334)).

Reply:
Based on the discussion we had before, maybe we don't even need the `sticky` property. We can still rely on the `duration` to make the toast be sticky. However, we need to keep compatibility with current API: we were using `0` to keep a toast displayed for undetermined time, and now the new snackbar uses `-1` for that.